tuist/tuist
# Remove CFBundleExecutable from iOS resource bundle target plists

Part of: #1360 

### Short description 📝

Currently generating the plist file for an iOS resource bundle target includes the key `CFBundleExecutable`, having it included causes a validation error when uploading application to the app store

```
Error 1: ITMS-90535 - Unexpected CFBundleExecutable Key
```

### Solution 📦

Omit that key incase the target is an iOS bundle.

### Implementation 👩‍💻👨‍💻


- [x] Update info plist provider


### Test Plan 🛠:

- Run `tuist generate` within `fixtures/ios_app_with_framework_and_resources/StaticFramework`
- Verify the generate info plist in `Derived` directory does not contain the `CFBundleExecutable` key
